### DISCUSSION TITLE
NCANode v3.2.1

### DISCUSSION THREAD

**Discussion by malikzh** [malikzh/NCANode#143] [General]
- Исправления касательно war билда для Tomcat v9.x

<hr /><em>This discussion was created from the release <a href='https://github.com/malikzh/NCANode/releases/tag/3.2.1'>NCANode v3.2.1</a>.</em>

**Comment by SoftLabNS** [comment]
![image](https://user-images.githubusercontent.com/61815611/199948388-c5a1db11-8a7f-4089-90eb-d8ccde4a56a7.png)

Добрый день. Вышла ошибка при компиляции (( Не поможете что нужно сделать?
![image](https://user-images.githubusercontent.com/61815611/199948574-a77e4ac9-970b-44fd-919c-7fefbb95ce66.png)

**Comment by malikzh** [comment]
Судя по скринам, вероятно у Вас на данный момент установлен JDK 11, а
NCANode использует JDK 17. Попробуйте собрать используя Oracle JDK 17:
https://www.oracle.com/java/technologies/downloads/#java17
или используя OpenJDK 17
https://jdk.java.net/archive/
или Amazon Corretto 17
https://aws.amazon.com/ru/corretto/?filtered-posts.sort-by=item.additionalFields.createdDate&filtered-posts.sort-order=desc

пт, 4 нояб. 2022 г. в 16:15, SoftLabNS ***@***.***>:

> [image: image]
> <https://user-images.githubusercontent.com/61815611/199948388-c5a1db11-8a7f-4089-90eb-d8ccde4a56a7.png>
>
> Добрый день. Вышла ошибка при компиляции (( Не поможете что нужно сделать?
> [image: image]
> <https://user-images.githubusercontent.com/61815611/199948574-a77e4ac9-970b-44fd-919c-7fefbb95ce66.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/malikzh/NCANode/discussions/143#discussioncomment-4055446>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIKJ3TIZQC2VMPKN6UK4TCLWGTO2JANCNFSM6AAAAAARGLUOCA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
С уважением, Malik Zharykov.
*Software Engineer*
E-mail: ***@***.***
GitHub: https://github.com/malikzh
Twitter: https://twitter.com/malikz_h
LinkedIn: https://www.linkedin.com/in/malik-zharykov-bba687165/

**Reply by SoftLabNS** [reply]
Спасибо огромное

**Comment by SoftLabNS** [comment]
Hello , хотелось бы выразить огромную благодарность разработчику системы. Супер проект. Вопрос: не хотите ли добавить поддержку gRPC?
